webpack.config.js

```diff
@@ -0,0 +1,23 @@
+var Encore = require('@symfony/webpack-encore');
+
+Encore
+    .setOutputPath('src/Resources/public/dist')
+    .setPublicPath('/bundles/sonataadmin/dist')
+    .setManifestKeyPrefix('dist')
+    .addEntry('sonata_admin', './src/Resources/public/js/sonata_admin.js')
+    .autoProvidejQuery()
+    .enableSassLoader()
+    .enableSingleRuntimeChunk()
+    .splitEntryChunks()
+    .enableSourceMaps(!Encore.isProduction())
+    .cleanupOutputBeforeBuild()
+    .enableBuildNotifications()
+    //.splitEntryChunks()
+    //.enableVersioning(Encore.isProduction())
+;
+
+let config = Encore.getWebpackConfig();
+var path = require('path');
+config.resolve.alias.jquery = path.join(__dirname, 'node_modules/jquery/dist/jquery');
```

Comment:
Can to upgrade version `@symfony/webpack-encore` and use method `addAliases` https://github.com/symfony/webpack-encore/blob/master/index.js#L357

Reply:
Nice thank you!